src/components/Calculator.vue

```diff
@@ -175,7 +178,8 @@
                     this.result();
 
                     const result = eval(this.calc_screen).toString();
-                    this.history.push(this.calc_screen + '=' + result);
+                    this.pushOperationInState(
+                        '<div class="expression">'+this.calc_screen+'</div><div class="equalSymbol">=</div><div class="expression">'+result+'</div>')
```

Comment:
It is bad idea to push html here. All html should be in template, in History page. Push `this.calc_screen` or if you want to style components (firstNumber, operation, secondNumber) you can push object.